README.md

```diff
@@ -3,9 +3,7 @@
 [![travis-develop][004]][005]
 [![npm-dependencies][006]][007]
 
-This is a Hapi plugin to load your sequelize models. Your models should be defined so that
```

Comment:
I think this headline could stay the same? guess it's still actual, that the models should be sequilize consumable and that sequlize is not a dependency?

Reply:
all right, sure 👍